WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -785,8 +792,12 @@ class AbstractPostListViewController: UIViewController, WPContentSyncHelperDeleg
 
         dismissAllNetworkErrorNotices()
 
-        let title = NSLocalizedString("Unable to Sync", comment: "Title of error prompt shown when a sync the user initiated fails.")
-        WPError.showNetworkingNotice(title: title, error: error)
+        if connectionAvailable() {
+            let title = NSLocalizedString("Unable to Sync", comment: "Title of error prompt shown when a sync the user initiated fails.")
```

Comment:
Why is this under if connectionAvailable()?

Reply:
The `connectionAvailable()` is used to check for _"Is there an internet connection?"_ and we decide how to show the Notice here.

Before this change, when the user is offline, the Notice was showing up with a generic _offline_ error message with an undesired "Unable to Sync" `title`. I changed this here so that if we are sure it's an offline error, we'll just use `presentNoNetworkAlert()` below. If it's something else like HTTP 404 or 500 then we use the `WPError.showNetworkingNotice` which creates user-friendly errors for those. 

Ideally, there should be one method to call but I'm not quite sure how to approach it yet.

Reply:
I mean shouldn't this be if !connectionAvailable() ? 
It seems a bit weird to me that we initialize the title for a no connection scenario under connection available check.

Reply:
Oh, so do you mean reversing the `if-else` condition? 

```swift 
        if !connectionAvailable() {
            presentNoNetworkAlert()
        } else {
            let title = NSLocalizedString("Unable to Sync", comment: "Title of error prompt shown when a sync the user initiated fails.")
            WPError.showNetworkingNotice(title: title, error: error)
        }
```

Reply:
Ok now I see. Would you mind adding a comment for that if to say something like if the call fails for any reason other than connectivity?

Reply:
Sure thing!

Reply:
Oh reversing it looks so much better!